packages/time-utilities/README.md

```diff
@@ -18,16 +18,14 @@
 -   [Description](#description)
 -   [Features](#features)
 -   [Installation](#installation)
--   [Usage](#usage)
-    -   [Basic Usage](#basic-usage)
 -   [Buy us some doughnuts](#buy-us-some-doughnuts)
 -   [Contributors ✨](#contributors-%E2%9C%A8)
 
 ## Description
 
 Working with Time and Duration can be a huge chore in any programming language. There are various time parsing libraries on the NPM registry but we are of the opinion that none of them meet the code quality that we desire to use. For this reason we have decided to make this package.
 
-Note that this package only re-exports everything from the four packages `@sapphire/cron`, `@sapphire/duration`, `@sapphire/timer-manager`, and `@sapphire/timestamp`. Those packages can also be installed individually if you only need a subset of the functionality.
+Note that this package only re-exports everything from the four packages `@sapphire/cron`, `@sapphire/duration`, `@sapphire/timer-manager`, and `@sapphire/timestamp`. Those packages should be cited individually for documentation and can be installed by themselves if you only need a subset of the functionality.
```

Comment:
Maybe we can link those dependencies so we don't need to copy and paste the names into the URL.